internal/controller/dependencyupdatecheck_controller.go

```diff
@@ -96,99 +137,32 @@ func (r *DependencyUpdateCheckReconciler) Reconcile(ctx context.Context, req ctr
 	}
 	dependencyupdatecheck.Annotations[MintMakerProcessedAnnotationName] = "true"
 
-	err = r.client.Update(ctx, dependencyupdatecheck)
+	err = r.Client.Update(ctx, dependencyupdatecheck)
```

Comment:
```suggestion
        patch := client.MergeFrom(dependencyupdatecheck.DeepCopy())
	err = r.Client.Patch(ctx, dependencyupdatecheck, patch)
```
I found the update doesn't work in my env, it reports error of:
```
Operation cannot be fulfilled on dependencyupdatechecks.appstudio.redhat.com \"dependencyupdatecheck-svjrb\": the object has been modified; please apply your changes to the latest version and try again
```
but I haven't found the root cause, but changing it to `Patch` works.